Admin/Admin.php

```diff
@@ -2926,7 +2926,7 @@ public function getAccessMapping()
     /**
      * {@inheritdoc}
      */
-    public function checkAccess($action, $object = null)
+    public function getAccess()
```

Comment:
Not sure if there is a need for this. The method returns a list of possible access names. If you want to check if an access exists, you could use ``hasAccess``.